build/sources/ceph-builders.upload

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+export OS_AUTH_URL="$OVH_AUTH_URL"
```

Comment:
I don't like much using OVH in the variables. I would rather re-use the same variable names.